utils/dot-remake.js

```diff
@@ -0,0 +1,55 @@
+const fs = require("fs");
+const path = require('path');
+const process = require('process');
+const nanoidGenerate = require('nanoid/generate');
+
+const log = console.log;
+
+function getUniqueId () {
+  return nanoidGenerate("1234567890abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ", 30);
+}
+
+function generateDotRemakeContent (multitenant) {
+  const dotRemakeContent = {
+    port: 3000,
+    sessionSecret: getUniqueId()
+  }
+  if (multitenant) {
+    dotRemakeContent.remakeMultiTenant = true;
+    dotRemakeContent.jwtSecret = nanoidGenerate("1234567890abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ", 30);
```

Comment:
I think we should use `crypto` here instead of `nanoid`. I've heard `nanoid` isn't supposed to be used for security tokens, just object ids and such.

Reply:
Agreed